manifests/i/IDRIX/VeraCrypt/1.25.4/IDRIX.VeraCrypt.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: IDRIX.VeraCrypt
+PackageVersion: 1.25.4
+PackageLocale: en-US
+Publisher: IDRIX
+PackageName: VeraCrypt
+License: Apache 2.0
+ShortDescription: VeraCrypt is a free open source disk encryption software for Windows, Mac OSX and Linux.

```

Comment:
```suggestion
ShortDescription: VeraCrypt is a free open source disk encryption software for Windows, Mac OSX and Linux.
Description: |
  VeraCrypt is a software for establishing and maintaining an on-the-fly-encrypted volume (data storage device). On-the-fly encryption means that data is automatically
  encrypted right before it is saved and decrypted right after it is loaded, without any user intervention. No data stored on an encrypted volume can be read (decrypted)
  without using the correct password/keyfile(s) or correct encryption keys. Entire file system is encrypted (e.g., file names, folder names, contents of every file,
  free space, meta data, etc). Files can be copied to and from a mounted VeraCrypt volume just like they are copied to/from any normal disk (for example, by simple
  drag-and-drop operations). Files are automatically being decrypted on the fly (in memory/RAM) while they are being read or copied from an encrypted VeraCrypt volume.
  Similarly, files that are being written or copied to the VeraCrypt volume are automatically being encrypted on the fly (right before they are written to the disk) in RAM.
  Note that this does not mean that the whole file that is to be encrypted/decrypted must be stored in RAM before it can be encrypted/decrypted. There are no extra memory
  (RAM) requirements for VeraCrypt.
Moniker: veracrypt
Tags: 
- encryption
- decryption
- hdd
- privacy
- security
- ssd
```